### PR TITLE
Add support for passing PHIDs

### PR DESCRIPTION
**Pull request by filippog**
@knyar this is a sketch of a solution for PHIDs, what do you think ? I'm not sure for example if changing the signature of `process_task` makes more sense vs looking up PHIDs before `process_task`.

### CODE REVIEW

File: phalerts.py
Comment (knyar) [inline phalerts.py]: `phids` seems to always be a list, so perhaps we no longer need the `list()` here?

**Review by knyar — APPROVED** [review]
This looks good. I agree that `process_task` is a better place to look up PHIDs.

Did you want to make any other changes before I merge this? If you have a minute to adjust README and document that projects can now be passed by name or by ID, I'd appreciate that.

Thanks!

File: phalerts.py
Comment (filippog) [reply phalerts.py]: Indeed, fixed!

**Comment by filippog** [comment]
> This looks good. I agree that `process_task` is a better place to look up PHIDs.
> 
> Did you want to make any other changes before I merge this? If you have a minute to adjust README and document that projects can now be passed by name or by ID, I'd appreciate that.

For sure! I've updated the README, let me know what you think!

I've un-drafted the PR, feel free to merge when/if it looks good now!

**Comment by knyar** [comment]
Thank you!

**Pull request merged**
